docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -1,107 +1,79 @@
-----
+==== Using the adapter
 
-If the `keycloak.json` file is in a different location you can specify it:
+The following example shows how to initialize the adapter, make sure to replace the options passed to the `Keycloak` constructor with those of the client you have configured.
 
 [source,javascript]
 ----
-const keycloak = new Keycloak('http://localhost:8080/myapp/keycloak.json');
-----
-
-Alternatively, you can pass in a JavaScript object with the required configuration instead:
+import Keycloak from 'keycloak-js';
 
-[source,javascript,subs="attributes+"]
-----
 const keycloak = new Keycloak({
     url: 'http://keycloak-server${kc_base_path}',
     realm: 'myrealm',
     clientId: 'myapp'
 });
+
+try {
+    const authenticated = await keycloak.init();
+    console.log(`User is ${authenticated ? 'authenticated' : 'not authenticated'}`);
+} catch (error) {
+    console.error('Failed to initialize adapter:', error);
+}
 ----
 
-By default to authenticate you need to call the `login` function. However, there are two options available to make the adapter automatically authenticate. You
-can pass `login-required` or `check-sso` to the init function. `login-required` will authenticate the client if the user is logged-in to {project_name}
-or display the login page if not. `check-sso` will only authenticate the client if the user is already logged-in, if the user is not logged-in the browser will be
-redirected back to the application and remain unauthenticated.
+By default to authenticate you need to call the `login` function. However, there are two options available to make the adapter automatically authenticate. You can pass `login-required` or `check-sso` to the `init()` function. `login-required` will authenticate the client if the user is logged-in to {project_name} or display the login page if not. `check-sso` will only authenticate the client if the user is already logged-in, if the user is not logged-in the browser will be redirected back to the application and remain unauthenticated.
 
-You can configure a _silent_ `check-sso` option.
-With this feature enabled, your browser won't do a full redirect to the {project_name} server and back to your application, but this action will be performed in a hidden iframe, so your application resources only need to be loaded and parsed once by the browser when the app is initialized and not again after the redirect back from {project_name} to your app.
-This is particularly useful in case of SPAs (Single Page Applications).
+You can configure a _silent_ `check-sso` option. With this feature enabled, your browser won't do a full redirect to the {project_name} server and back to your application, but this action will be performed in a hidden iframe, so your application resources only need to be loaded and parsed once by the browser when the app is initialized and not again after the redirect back from {project_name} to your app. This is particularly useful in case of SPAs (Single Page Applications).
```

Comment:
```suggestion
You can configure a _silent_ `check-sso` option. With this feature enabled, your browser will not perform a full redirect to the {project_name} server and back to your application, but this action will be performed in a hidden iframe. Therefore, your application resources are only loaded and parsed once by the browser when the application is initialized and not again after the redirect back from {project_name} to your application. This approach is particularly useful in case of SPAs (Single Page Applications).
```